compiler/rustc_symbol_mangling/src/lib.rs

```diff
@@ -245,7 +245,11 @@ fn compute_symbol_name(
     // 2. we favor `instantiating_crate` where possible (i.e. when `Some`)
     let mangling_version_crate = instantiating_crate.unwrap_or(def_id.krate);
     let mangling_version = if mangling_version_crate == LOCAL_CRATE {
-        tcx.sess.opts.debugging_opts.symbol_mangling_version
+        tcx.sess
+            .opts
+            .debugging_opts
+            .symbol_mangling_version
+            .unwrap_or(SymbolManglingVersion::default())
```

Comment:
Same thing here, I could replace this with a function on `DebuggingOptions` to instead allow calling:

```rust
    ...debugging_opts.get_symbol_mangling_version()
```

Reply:
(by `Same thing here`, I'm referring to the next comment, which came in out of order)

Reply:
I'm making this change in a new commit